lib/const.rb

```diff
@@ -63,13 +63,15 @@
 case ARCH
 when 'aarch64', 'armv7l'
   CREW_TGT = 'armv7l-cros-linux-gnueabihf'
+  CREW_BUILD = 'armv7l-cros-linux-gnueabihf'
 when 'i686'
   CREW_TGT = 'i686-cros-linux-gnu'
+  CREW_BUILD = 'i686-cros-linux-gnu'
 when 'x86_64'
   CREW_TGT = 'x86_64-cros-linux-gnu'
+  CREW_BUILD = 'x86_64-cros-linux-gnu'
```

Comment:
So now we have 2 constants that are the same?  Can we just revert back to only one?

Reply:
I do want to leave because I'm planning on filing an upstream bug with GNU automake about the issue. This is just temporary ;)